src/app/components/universal-tabbed-card/universal-tabbed-card.component.ts

```diff
@@ -0,0 +1,55 @@
+import { AfterContentInit, Component, ContentChildren, QueryList } from '@angular/core';
+import { UniversalTabComponent } from './universal-tab/universal-tab.component';
```

Comment:
should have a blank space for line 2 - want to separate package imports and local imports